unicodetools/src/main/java/org/unicode/props/UcdPropertyValues.java

```diff
@@ -95,6 +95,7 @@ public static Age_Values forName(String name) {
         }
     }
 
+    // Arabic_Shaping_Schematic_Name
```

Comment:
What is this comment supposed to do?

Reply:
This is a generated file; it has a comment for each property that is not enumerated.